service-commands/src/commands/seed/cliToCommandMap.js

```diff
@@ -48,184 +46,187 @@ Notes on params:
 */
 
```

Comment:
are there any changes in this file besides lint?

Reply:
nope